src/pymovements/plotting/main_sequence_plot.py

```diff
@@ -24,25 +24,39 @@
 
 import matplotlib.pyplot as plt
 import polars as pl
+from matplotlib.collections import Collection
 from polars import ColumnNotFoundError
 
 from pymovements.events import EventDataFrame
 
 
 def main_sequence_plot(
         event_df: EventDataFrame,
+        marker_size: float = 25,
+        color: str = 'purple',
+        alpha: float = 0.5,
+        marker: str = 'o',
         figsize: tuple[int, int] = (15, 5),
         title: str | None = None,
         savepath: str | None = None,
         show: bool = True,
+        **kwargs: Collection,
```

Comment:
I'm not sure if I understand the typehint here, but if mypy is happy, I am also happy